receiver/mongodbreceiver/metadata.yaml

```diff
@@ -238,3 +256,39 @@ metrics:
       aggregation: cumulative
       monotonic: false
     attributes: []
+  mongodb.locks.acquire_count:
+    description: Number of times the lock was acquired in the specified mode.
+    unit: "{count}"
+    enabled: true
+    sum:
+      value_type: int
+      aggregation: cumulative
+      monotonic: false
+    attributes: [database, lock_type, lock_mode]
+  mongodb.locks.acquire_wait_count:
+    description: Number of times the lock acquisitions encountered waits because the locks were held in a conflicting mode.
+    unit: "{count}"
+    enabled: true
+    sum:
+      value_type: int
+      aggregation: cumulative
+      monotonic: false
+    attributes: [database, lock_type, lock_mode]
+  mongodb.locks.time_acquiring_micros:
+    description: Cumulative wait time in microseconds for the lock acquisitions.
+    unit: "ms"
+    enabled: true
+    sum:
+      value_type: int
+      aggregation: cumulative
+      monotonic: false
+    attributes: [database, lock_type, lock_mode]
+  mongodb.locks.deadlock_count:
```

Comment:
```suggestion
  mongodb.lock.deadlock.count:
```

Reply:
Updated the metric name as per suggestion